app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,40 @@
+class CommentsController < ApplicationController  
+  
+  def create
+    company = Company.find(params[:company_id])
+    job = company.jobs.find(params[:job_id])
+    job.comments.create(comment_params)
```

Comment:
don't forget to handle the errors. what happens if a user enters something incorrectly and the comment doesn't get created?